salesforce/latest/install/index.md

```diff
@@ -237,13 +241,13 @@ Make sure that you've applied your AMP files, downloaded the Alfresco Content Co
 
 2. Under **PLATFORM TOOLS**, click **Apps > App Manager**.
 
-3. On the **Lightning Experience App Manager** screen, click the down-arrow icon for the app that you created in [Step 3: Configure the app in Salesforce](#configappinsalesforce-lightning) and select View.
+3. On the **Lightning Experience App Manager** screen, click the down-arrow icon for the app that you created in [Step 3: Configure the app in Salesforce](#configappinsalesforce-lightning) and select **View**.
 
     ![sf-view]({% link salesforce/images/sf-view.png %})
 
-    In the **API (Enable OAuth Settings)** section, you'll see entries for the consumer key and consumer secret.
+4. In the **API (Enable OAuth Settings)** section, click **Manage Consumer Details** to reveal the Consumer Key and Secret. You'll return to this screen to copy the code for both fields.
 
```

Comment:
```suggestion
In the **API (Enable OAuth Settings)** section, click **Manage Consumer Details** to reveal the Consumer Key and Secret. You'll return to this screen to copy the code for both _the_ fields.
```

Reply:
Thanks for the suggestion @Prosune but I'm not carrying it forward in this instance.